src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -2042,6 +2044,8 @@ def save_fuzz_targets(output):
   """Saves fuzz targets that were seen in the build to the database."""
   if not output.fuzz_task_output.fuzz_targets:
     return
+
+  logs.log(f'Saving fuzz targets: {output.fuzz_task_output.fuzz_targets}.')
```

Comment:
I think `save_fuzz_targets` should be called before calling error_handling in `utask_postprocess`.

Reply:
I'm not sure it will have any practical imapct and I'm generally opposed to running any code before error handling.